ArcTouchChallenge/ArcTouchChallenge/Model/Movie/MovieDetail.swift

```diff
@@ -0,0 +1,143 @@
+//
+//  MovieDetail.swift
+//
+//  Created by Arthur Augusto Sousa Marques on 3/14/18
+//  Copyright (c) . All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+class MovieDetail: Model {
+
+  // MARK: Declaration for string constants to be used to decode and also serialize.
+  private struct SerializationKeys {
+    static let budget = "budget"
+    static let backdropPath = "backdrop_path"
+    static let revenue = "revenue"
+    static let voteCount = "vote_count"
+    static let overview = "overview"
+    static let voteAverage = "vote_average"
+    static let video = "video"
+    static let imdbId = "imdb_id"
+    static let id = "id"
+    static let title = "title"
+    static let homepage = "homepage"
+    static let productionCompanies = "production_companies"
+    static let belongsToCollection = "belongs_to_collection"
+    static let posterPath = "poster_path"
+    static let adult = "adult"
+    static let genres = "genres"
+    static let spokenLanguages = "spoken_languages"
+    static let status = "status"
+    static let runtime = "runtime"
+    static let originalTitle = "original_title"
+    static let releaseDate = "release_date"
+    static let originalLanguage = "original_language"
+    static let popularity = "popularity"
+    static let tagline = "tagline"
+    static let productionCountries = "production_countries"
+  }
+
+  // MARK: Properties
+  public var budget: Int?
+  public var backdropPath: String?
+  public var revenue: Int?
+  public var voteCount: Int?
+  public var overview: String?
+  public var voteAverage: Float?
+  public var video: Bool? = false
```

Comment:
This was created automatically from the component third party and it is optional because the service can be changed and can not return its value, and without that a unwrapped object error would occur.